package.json

```diff
@@ -2,18 +2,21 @@
 	"name": "uc-integration-api",
 	"version": "0.1.0",
 	"description": "NodeJS wrapper for the Unfolded Circle Integration API",
-	"main": "index.js",
+	"main": "dist/index.js",
+	"type": "module",
 	"directories": {
 		"example": "examples",
 		"lib": "lib"
 	},
 	"dependencies": {
+		"bonjour": "^3.5.0",
```

Comment:
Do not include this old 8 year old abandoned package!
We are using bonjour-service, which is a modern rewrite in TypeScript.

Reply:
I deleted this package.